### PR TITLE
Sandbox-classic: revert back the change of default contract ID seeding

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
We revert the change of default contract ID seeding that was
introduced in #7945.

CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: language-support/java/codegen/src/ledger-tests/scala/com/digitalasset/TestUtil.scala [hunk 1]
```diff
@@ -48,7 +48,6 @@ object TestUtil {
   )(implicit resourceContext: ResourceContext): Future[Assertion] = {
     val config = sandbox.DefaultConfig.copy(
       port = Port.Dynamic,
-      seeding = None,
```
Comment (cocreature) [inline language-support/java/codegen/src/ledger-tests/scala/com/digitalasset/TestUtil.scala:51]: this seems independent of this right?

**Review by cocreature — APPROVED** [review]
Thanks for fixing this so quickly!

**Review by SamirTalwar-DA — APPROVED** [review]

**Comment by SamirTalwar-DA** [comment]
For context: I think I broke this because I thought I was changing test code. I did not intend to change the behavior of Sandbox in production, only in testing.

File: language-support/java/codegen/src/ledger-tests/scala/com/digitalasset/TestUtil.scala (hunk 1 above)
Comment (remyhaemmerle-da) [reply language-support/java/codegen/src/ledger-tests/scala/com/digitalasset/TestUtil.scala:51]: I revert the two last commits of #7945. This is the last one that overrides the `seeding = Some(Seeding.Weak)`  from the culprit commit. I think it is fair to revert this as well.

Comment (cocreature) [reply language-support/java/codegen/src/ledger-tests/scala/com/digitalasset/TestUtil.scala:51]: Makes sense :+1:

**Pull request merged**